easybuild/easyconfigs/g/GCCcore/GCCcore-12.3.0.eb

```diff
@@ -36,7 +36,7 @@ patches = [
     'GCCcore-12.1.0_fix-double-destruct.patch',
     'GCCcore-12.2.0_fix-avx512-misoptimization.patch',
     'GCCcore-12.2.0_improve-cuda-compatibility.patch',
-    'GCCcore-12.x_riscv_multiarch_support.patch',
```

Comment:
Removing this patch does not seem intentional. Perhaps you added the patch to an outdated GCCcore easyconfig?

Reply:
Crap, you're right. I did a `--copy-ec`, but I guess this got added since the latest release. I'll add it again.